crates/environ/src/module_environ.rs

```diff
@@ -369,6 +369,11 @@ impl<'data> cranelift_wasm::ModuleEnvironment<'data> for ModuleEnvironment<'data
         });
         Ok(())
     }
+
+    fn declare_func_name(&mut self, func_index: FuncIndex, name: &'data str) -> WasmResult<()> {
```

Comment:
Is this used somewhere I'm not seeing?

Reply:
cranelift provides names for function (currently it comes from name section)

Reply:
Ok, but where is this being called?

Reply:
Oh sorry, I didn't see this was in a trail impl.  Never mind!